src/main/java/io/wisetime/connector/jira/JiraConnector.java

```diff
@@ -191,11 +197,8 @@ public PostResult postTime(final Request request, final TimeGroup timeGroup) {
 
     try {
       jiraDao.asTransaction(() ->
-          timeGroup
-              .getTags()
+          relevantTags
               .stream()
-              .filter(relevantProjectKey)
```

Comment:
`jiraDao.asTransaction(...)` will fail if there' no any query is executed. Thus, I moved the checking of relevant tags outside the transaction block.